src/main/java/patio/voting/services/internal/VotingSchedulingService.java

```diff
@@ -90,7 +89,7 @@ public VotingSchedulingService(
   }
 
   @Override
-  @Scheduled(fixedRate = "30s", initialDelay = "30s")
+  // @Scheduled(fixedRate = "30s", initialDelay = "30s")
```

Comment:
We shouldn't forget to remove this comment before merging

Reply:
changed